### PR TITLE
Wording changes

### PR DESCRIPTION
**Pull request by LisandroFernandezSF**
### What does this PR do?
Gathering wording changes
### What issues does this PR fix or reference?
@W-8247907@ @W-8517984@

### CODE REVIEW

File: packages/salesforcedx-vscode-lwc/src/messages/i18n.ts
Comment (jag-sfdc) [inline packages/salesforcedx-vscode-lwc/src/messages/i18n.ts:23]: Looks like it is still pointing to the beta docs. Can you get the update the link from @ekapner

Comment (ekapner) [reply packages/salesforcedx-vscode-lwc/src/messages/i18n.ts:23]: @LisandroFernandezSF - this will be the link when the content goes live in mid January:
https://developer.salesforce.com/tools/vscode/en/codepreview/lwccodepreview

**Review by ekapner — APPROVED** [review]
LGTM

**Review by sfsholden — APPROVED** [review]

**Comment by sfsholden** [comment]
Should we move W-8517984 to waiting until it's time for this PR to be merged?

**Comment by randi274** [comment]
@jag-sfdc do you know if this can/should be merged? If not, we should go ahead and close this one.

**Comment by gbockus-sf** [comment]
Look no longer valid.

**Pull request closed without merging**
